### PR TITLE
ACMS-3628: Refactor test for site studio.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_site_studio/tests/src/ExistingSite/SiteStudioCoreTest.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    $assert_session->elementExists('css', '.acquia-cms-site-studio-core-form');
+
+    // Assert that Save & Ignore buttons are present on form.
+    $assert_session->buttonExists('Save');
+    $assert_session->buttonExists('Ignore');
+    $assert_session->linkExists('Advanced');
+
+    // Assert that the expected fields show up.
+    $assert_session->fieldExists('API key');
+    $assert_session->fieldExists('Agency key');
```
Comment (rajeshreeputra) [inline modules/acquia_cms_site_studio/tests/src/ExistingSite/SiteStudioCoreTest.php:53]: Move this test to Functional as we are not saving the form.

**Review by rajeshreeputra — CHANGES_REQUESTED** [review]
Changes looks good

File: modules/acquia_cms_site_studio/tests/src/ExistingSite/SiteStudioCoreTest.php (hunk 1 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_site_studio/tests/src/ExistingSite/SiteStudioCoreTest.php]: Done.

**Comment by deepakmishra2** [comment]
@chandan-singh7929 - i think we should push this change over to DO to verify if that Gitlab CI pipeline is passing.

cc @vishalkhode1

**Comment by chandan-singh7929** [comment]
> @chandan-singh7929 - i think we should push this change over to DO to verify if that Gitlab CI pipeline is passing.
> 
> cc @vishalkhode1

This is already verified locally with orca template and all tests are passing.

**Review by rajeshreeputra — APPROVED** [review]

**Review by apathak18 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**